api/net/Areas/Subscriber/Controllers/WorkOrderController.cs

```diff
@@ -108,34 +112,16 @@ public IActionResult Find()
     [ProducesResponseType(typeof(WorkOrderMessageModel), (int)HttpStatusCode.OK)]
     [ProducesResponseType((int)HttpStatusCode.NoContent)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
-    [SwaggerOperation(Tags = new[] { "Content" })]
+    [SwaggerOperation(Tags = new[] { "WorkOrder" })]
     public async Task<IActionResult> RequestTranscriptionAsync(long contentId)
     {
-        var content = _contentService.FindById(contentId) ?? throw new InvalidOperationException("Content does not exist");
-        if (String.IsNullOrWhiteSpace(_kafkaOptions.TranscriptionTopic)) throw new ConfigurationException("Kafka transcription topic not configured.");
-
-        // Only allow one work order transcript request at a time.
-        // TODO: Handle blocked work orders stuck in progress.
-        var workOrders = _workOrderService.FindByContentId(contentId);
-        if (workOrders.Any(o => o.WorkType == WorkOrderType.Transcription && _workLimiterStatus.Contains(o.Status)))
-            return new JsonResult(new WorkOrderMessageModel(workOrders.First(o => o.WorkType == WorkOrderType.Transcription && _workLimiterStatus.Contains(o.Status)), _serializerOptions))
+        var workOrder = await _workOrderHelper.RequestTranscriptionAsync(contentId, true);
```

Comment:
Use common code so that bugs like this don't occur in one part of the application and not in another.